rever/activities/appimage.xsh

```diff
@@ -18,7 +18,9 @@ class AppImage(Activity):
     def _func(self, template='$VERSION'):
         if not self.appimage_descr_dir.exists():
             return None
-
+        
+        python_ver = ${...}.get('APPIMAGE_PYTHON_VERSION', None)
```

Comment:
This should be in the function signature as `python_version=None`. It will then automatically be bound to `$APPIMAGE_PYTHON_VERSION`.  Please take a look at how other activities implement this, for example the PyPI activity on this line: https://github.com/regro/rever/blob/master/rever/activities/pypi.xsh#L109

Reply:
Indeed! Thanks!